esx_service/utils/auth_data.py

```diff
@@ -410,7 +422,8 @@ def create_default_tenant(self):
                                            name=auth.DEFAULT_TENANT, 
                                            description="This is a default tenant", 
                                            vms=[], 
-                                           privileges=[])
+                                           privileges=[],
+                                           tenant_uuid=auth.DEFAULT_TENANT_UUID)
```

Comment:
I don't think it is a good approach. Basically, what I am doing is create_tenant has a new parameter, "tenant_uuid", if caller choose to parse this param, we will create a tenant using the passed "tenant_uuid", otherwise, we will auto-generate a uuid for this tenant. Caller of "create_tenant" should has more knowledge and more control for this.